app/Http/Controllers/FailedJobController.php

```diff
@@ -30,7 +31,7 @@ protected function addParsedPropertiesToFailedJob($failedJob)
         $failedJob->commandName = $json->data->commandName;
         $failedJob->errorMessage = Str::limit($failedJob->exception, 255);
 
-        if ($failedJob->commandName === 'Chompy\Jobs\CreateCallPowerPostInRogue') {
+        if (Str::contains($failedJob->commandName, 'CallPower') || Str::contains($failedJob->commandName, 'SoftEdge')) {
```

Comment:
Now that there's a couple of these, it might make sense to create an interface:

```php
interface Inspectable {
    /**
     * Returns the parameters passed to this job.
     *
     * @return array
     */
    public function getParameters();
}
```

```php
class CreateSoftEdgePostInRogue implements ShouldQueue, Inspectable
{
    // ...
```

Then, we can check here whether the failed job's class implements this method:

```php
if (array_has(class_implements($failedJob->commandName), Inspectable::class)) {
   // Now we know that $command will have a getParameters method!
}
```

Reply:
I'm into it - where's a good spot for the interface to live? `/App/Jobs/Inspectable.php`?

Reply:
Yeah, I think that works well. 👍

Reply:
I had some trouble getting this to work... will give it a shot in another PR